GPU/Vulkan/VulkanUtil.cpp

```diff
@@ -85,14 +85,14 @@ void VulkanComputeShaderManager::InitDeviceObjects() {
 
 	std::vector<VkDescriptorPoolSize> dpTypes;
 	dpTypes.resize(2);
-	dpTypes[0].descriptorCount = 8192;
+	dpTypes[0].descriptorCount = 4096;
```

Comment:
I feel like we've actually hit these limits though and that's why they're kinda high now?

-[Unknown]

Reply:
I guess, they just seemed a bit excessive. But I'll restore for now.